### PR TITLE
Improved Search Results Page UI and refactored code

### LINKED ISSUE #15961: Non intuitive UI on search Results Page
The UI is non-intuitive, links/buttons appear as texts.  We can replace those text with proper button to toggle results.
image

### PR DESCRIPTION
**Pull request by thesmallstar**
Signed-off-by: Manthan Surkar <manthan.surkar@gmail.com>
Initially, Two divs were used, which was unnecessary. I have made the required changes for code and UI.
fixes #15961

New UI:
image

image

### CODE REVIEW

**Comment by thesmallstar** [comment]
image

File: templates/database/search/main.twig [hunk 1]
```diff
@@ -1,4 +1,9 @@
 <a id="db_search"></a>
+
+<div id="togglesearchOptionsdiv">
+    <br>
```
Comment (mauriciofauth) [inline templates/database/search/main.twig]: Only use `<br>` for paragraph breaks. Please remove it.

File: templates/database/search/main.twig [hunk 2]
```diff
@@ -1,4 +1,9 @@
...
+    <button class="btn btn-primary" id="togglesearchformlink"></a>
```
Comment (mauriciofauth) [inline templates/database/search/main.twig]:
```suggestion
    <button class="btn btn-primary" id="togglesearchformlink"></button>
```

**Review by MauricioFauth — CHANGES_REQUESTED** [review]

**Comment by williamdes** [comment]
@thesmallstar could you fix this PR please ?

**Pull request closed without merging**